Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/ReadBlob.cs

```diff
@@ -11,42 +11,42 @@ namespace Frends.AzureBlobStorage.ReadBlob
     public class AzureBlobStorage
     {
         /// <summary>
-        /// Read a single file to Azure blob storage using connection string or SAS Token.
-        /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.AzureBlobStorage.ReadBlob)
-        /// <param name="source">Information about which Blob to read.</param>
-        /// <returns>Object { string content }</returns>
+        /// Read a single file from Azure Storage.
         /// </summary>
+        /// <param name="source">Source connection parameters.</param>
+        /// <param name="options">Options for the task</param>
+        /// <returns>object { string Content }</returns>
 
-        public static Result ReadBlob([PropertyTab] Source source, CancellationToken cancellationToken)
+        public static Result ReadBlob([PropertyTab] Source source, [PropertyTab] Options options, CancellationToken cancellationToken)
         {
-            var data = ReadBlobContent(source, cancellationToken);
-            return new Result(data);
+            var blob = CreateBlobClient(source);
+            var result = blob.DownloadContent(cancellationToken).Value;
```

Comment:
You should use async method here to make sure that the operation has really finished before getting the value.